src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -13,13 +13,21 @@ namespace System.Net.Http
 {
     public partial class HttpClientHandler : HttpMessageHandler
     {
+#if TARGETS_BROWSER
+        private readonly BrowserHttpHandler _socketsHttpHandler;
```

Comment:
Having a "_socketsHttpHandler" field that's not a "SocketsHttpHandler" is confusing.  It'd be better to rename this field (both declarations of it) "_underlyingHandler" or something along those lines.

Reply:
Addressed in recent commits